src/App.js

```diff
@@ -1,19 +1,28 @@
 import './App.css';
 import { Provider } from 'react-redux';
-import store from './redux/store';
+import { ThemeProvider, createTheme } from '@material-ui/core/styles';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import { ThemeProvider, createTheme } from "@material-ui/core/styles";
```